Jantimizer/QueryParser/Models/JoinNode.cs

```diff
@@ -8,17 +8,32 @@ namespace QueryParser.Models
 {
     public class JoinNode : INode
     {
+        public enum ComparisonType {
+            None,
+            Equal,
+            Less,
+            More,
+            EqualOrLess,
+            EqualOrMore
+        };
+
         public int Id { get; internal set; }
+        public ComparisonType ComType { get; internal set; }
         public string LeftTable { get; internal set; }
+        public string LeftAttribute { get; internal set; }
         public string RightTable { get; internal set; }
+        public string RightAttribute { get; internal set; }
         public string JoinCondition { get; internal set; }
         public List<string> ConditionTables { get; }
 
-        public JoinNode(int id, string leftTable, string rightTable, string joinCondition)
+        public JoinNode(int id, ComparisonType type, string leftTable, string leftAttribute, string rightTable, string rightAttribute, string joinCondition)
```

Comment:
Does this not assume that there can ONLY be two attributes at most in a condition? Would the `AND` and `OR` words not be ignored or crash with this? (I think it was able to parse those keywords too before)

Reply:
I'd rather do that in a separate pull request